### DISCUSSION TITLE
Free Organization Account Collaborator

### DISCUSSION THREAD

**Discussion by ibahnasy** [community/community#23458] [New to GitHub]
<p>Can a collaborator delete a branch or the whole repository in a free Github organization account?</p>

**Comment by mpboom** [comment]
<p>For branches: yes, if they have been given permission to do so. Entire repositories: no, since you need to be an organization member with appropriate permissions - and you seem to be talking about collaborators (which is a per-repository thing, a collaborator does not have to be an organization member).</p>

**Comment by ibahnasy** [comment]
<p>Thank you for your reply.<br>
What what is the best way to protect a branch from deletion and push events for a free organization account while working with organization members or collaborators?</p>
<p>Developers should be able to read, create new branches, and making PR.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Use <a href="https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches" rel="noopener nofollow ugc">branch protection rules</a>.</p>
